app/controllers/state_file/questions/questions_controller.rb

```diff
@@ -73,8 +73,6 @@ def path_for_step(step)
       # by default, most state file questions have no illustration
       def illustration_path; end
 
-      def card_postscript; end
```

Comment:
should we go around removing all of these since the view does first check `respond_to?`?